crates/iceberg/Cargo.toml

```diff
@@ -83,5 +83,6 @@ ctor = { workspace = true }
 iceberg-catalog-memory = { workspace = true }
 iceberg_test_utils = { path = "../test_utils", features = ["tests"] }
 pretty_assertions = { workspace = true }
+rand = "0.8"
```

Comment:
Sure, will do. I was wondering, what exactly is our policy on this as I've not seen it written anywhere? Is it simply for all dependencies to have their details at the workspace level, and for package level cargo toml to only contain workspace = true? Or is there more to it? 

It might be helpful to add a comment in the cargo.toml for clarity.

Reply:
Moved to workspace anyway